project.rb

```diff
@@ -0,0 +1,15 @@
+require 'faker'
+
+class Project
+
+  attr_reader :project_name, :identifier, :new_version_name
```

Comment:
too complicated properties' titles
if you use `name` instead of `project_name` you'll be able to call this property like this:
`project.name`
instead of this
`project.project_name`
the same with version:
use `project.version`, please